apps/cookbook/src/app/showcase/showcase.component.scss

```diff
@@ -66,7 +66,7 @@ h4 {
   fill: currentColor;
 }
 
-::ng-deep .cookbook-properties {
+::ng-deep .cookbook-members-table {
```

Comment:
I believe this could be moved to the individual table-comps now? 🤔 
(It's generally considered bad practice to use `::ng-deep`)